src/script/auth/module/action/AuthAction.ts

```diff
@@ -107,11 +107,10 @@ export class AuthAction {
         await dispatch(selfAction.fetchSelf());
         let entropyData: Uint8Array | undefined = undefined;
         if (getEntropy) {
+          entropyData = await getEntropy();
           try {
-            await core.loadAndValidateLocalClient();
-          } catch (e) {
-            entropyData = await getEntropy();
-          }
+            await core.loadAndValidateLocalClient(entropyData);
+          } catch (e) {}
```

Comment:
i dont think you should change this, it is set up like this for a reason

Reply:
i need the entropy to pass to the core function now. So I need to fetch it before the try statement

Reply:
I don't think you need an entropy source when you load the local client, it already has generated it's identity.  Only when you create a new one ou should could need custom entropy (just like it's done right now with Proteus devices). Or are there specific measures when loading created client ?

Reply:
I just assumed that if you generated the client with the entropy, you will need to load it again with it to get the same identifier in the end.

Reply:
The webapp part is already implemented and the entropy is already given to the core, so the API is already there and existing (also the condition to display the entropy drawing field on windows). 
So here the change should only be passing thos entropy (that the webapp already gives to the core) to core-crypto and that should do the trick :)